zaas-client/src/main/java/org/zowe/apiml/zaasclient/service/internal/PassTicketServiceImpl.java

```diff
@@ -78,7 +78,7 @@ private String extractPassTicket(CloseableHttpResponse response) throws IOExcept
             } else if (statusCode == 400) {
                 throw new ZaasClientException(ZaasClientErrorCodes.BAD_REQUEST, obtainedMessage);
             } else if (statusCode == 500) {
-                throw new ZaasClientException(ZaasClientErrorCodes.SERVICE_UNAVAILABLE, obtainedMessage);
+                throw new ZaasClientException(ZaasClientErrorCodes.INVALID_REQUEST_AUTHENTICATION, obtainedMessage);
```

Comment:
Are we sure it is only one possible error response?

Reply:
no, but most of error codes from AbstractIRRPassTicketException are based on authentication issues and they all return 500. so I thought of making it INVALID_REQUEST_AUTHENTICATION.